core/src/server/rpc_module.rs

```diff
@@ -521,6 +783,16 @@ impl<Context> From<RpcModule<Context>> for Methods {
 }
 
 impl<Context: Send + Sync + 'static> RpcModule<Context> {
+	/// Return a mutable reference to the currently registered methods.
+	pub fn methods_mut(&mut self) -> &mut Methods {
```

Comment:
We don't want to expose this, can you remove it or make it private?

It's doesn't seem to used.